ui/utilities/common-newport/_index.scss

```diff
@@ -1,3 +1,8 @@
+body {
```

Comment:
This is a global rule to fix the body... Aren't we already setting the margin to 0? Either way I think this rule shouldn't be here.